blade-graphics/src/vulkan/init.rs

```diff
@@ -143,6 +150,9 @@ unsafe fn inspect_adapter(
     let name = ffi::CStr::from_ptr(properties.device_name.as_ptr());
     log::info!("Adapter {:?}", name);
 
+    let vendor_id = properties2_khr.properties.vendor_id;
```

Comment:
if we are doing this, let's also use `vendor_id` when computing `intel_unable_to_present` consistently